src/__tests__/disableRanges-test.js

```diff
@@ -281,6 +281,29 @@ test("Nesting disabledRanges", t => {
   )
   planCount += 1
 
+  testDisableRanges(
+    `/* stylelint-disable */
+    /* stylelint-enable bar */
+    /* stylelint-enable foo */
+    /* stylelint-disable foo */
+    /* stylelint-enable bar */
+    /* stylelint-enable */`,
```

Comment:
This looks to me like it's buggy. Do you agree? The rule `bar` should be *enabled* starting at line 2 (286) and not disabled again. And at line 6 (289) shouldn't there be an error?